README.md

```diff
@@ -133,6 +133,93 @@ Additional options:
 * `database`: Initial database to select. Defaults to SQL Server user profile settings.
 * `ssl`: Whether to use transport-level encryption for the entire SQL server traffic, defaults to `false`.
 
+### Data Type Mapping 
+
+[Microsoft Sql Server][m] type mapping reference. This reference table shows encoding/decoding types from [Microsoft Sql Server][m] types to java data types.
+
+
+| Microsoft SQL Server Type               | Supported Data Type                                                                                                              | 
+|:----------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------|
+| [bit][sql-bit-ref]                      | [Boolean*][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [tinyint][sql-all-int-ref]              | [Boolean][java-boolean-ref], [Byte*][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [smallint][sql-all-int-ref]             | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short*][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [int][sql-all-int-ref]                  | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer*][java-integer-ref], [Long][java-long-ref] |
+| [bigint][sql-all-int-ref]               | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long*][java-long-ref] |
+| [float][sql-float-real-ref]             | [Double*][java-double-ref], [Float][java-float-ref] 
+| [real][sql-float-real-ref]              | [Double][java-double-ref], [Float*][java-float-ref]   
+| [decimal][sql-decimal-ref]              | [BigDecimal][java-bigdecimal-ref] 
+| [numeric][sql-decimal-ref]              | [BigDecimal][java-bigdecimal-ref]
+| [uniqueidentifier][sql-uid-ref]         | [UUID*][java-uuid-ref], [String][java-string-ref]   
+| [smalldatetime][sql-smalldatetime-ref]  | [LocalDateTime][java-ldt-ref] 
+| [datetime][sql-datetime-ref]            | [LocalDateTime][java-ldt-ref] 
+| [datetime2][sql-datetime2-ref]          | [LocalDateTime][java-ldt-ref] 
+| [date][sql-date-ref]                    | [LocalDate][java-ld-ref] 
+| [time][sql-time-ref]                    | [LocalTime][java-time-ref] 
+| [timestamp][sql-timestamp-ref]          | byte[]
+| [datetimeoffset][sql-dtof-ref]          | [OffsetDateTime][java-odt-ref], [ZonedDateTime][java-zdt-ref]  
+| [smallmoney][sql-money-ref]             | [BigDecimal][java-bigdecimal-ref]
+| [money][sql-money-ref]                  | [BigDecimal][java-bigdecimal-ref]
+| [char][sql-(var)char-ref]               | [String][java-string-ref]
+| [varchar][sql-(var)char-ref]            | [String][java-string-ref]
+| [varcharmax][sql-(var)char-ref]         | ---
```

Comment:
We should probably explicitly say "not supported" to make things clear instead of `---`.